packages/vm/lib/evm/opFns.ts

```diff
@@ -713,7 +718,11 @@ export const handlers: { [k: string]: OpHandler } = {
     if (!value.isZero()) {
       runState.eei.useGas(new BN(runState._common.param('gasPrices', 'callValueTransfer')))
     }
-    gasLimit = maxCallGas(gasLimit, runState.eei.getGasLeft())
+    gasLimit = maxCallGas(gasLimit, runState.eei.getGasLeft(), runState)
+    // note that TW or later this cannot happen (it could have ran out of gas prior to getting here though)
```

Comment:
Won't make this a blocker either, but such abbreviations like `TW` are suboptimal since hard to read (I didn't make it at all to "Tangerine Whistle" on first round), better to fully write stuff like this down.

Reply:
Good point, will keep it in mind 👍